Spawn-App-iOS-SwiftUI/Views/SubComponents/NewTagsSubViews/Tags/FriendRow.swift

```diff
@@ -15,19 +15,31 @@ struct FriendRow: View {
         HStack {
             if let profilePictureString = friend.profilePicture {
                 Image(profilePictureString)
-                    .ProfileImageModifier(imageType: .chatMessage)
-                    .font(.system(size: 20))
+                    .resizable()
```

Comment:
Would've preferred if there was a way to modify `ProfileImageModifier.swift` instead of doing these custom attributes, but it's fine.